applications/asset_tracker_v2/src/modules/gps_module.c

```diff
@@ -454,6 +454,18 @@ static void search_start(void)
 		}
 	}
 
+	uint8_t use_case = NRF_MODEM_GNSS_USE_CASE_MULTIPLE_HOT_START;
+
+#if defined(CONFIG_GPS_MODULE_AGPS_FILTERED)
+	err = nrf_cloud_agps_configure_filter(use_case, CONFIG_GPS_MODULE_ELEVATION_MASK);
```

Comment:
Could you make a separate static function in `gps_module.c` that replaces `nrf_cloud_agps_configure_filter`?
So that we avoid a dependency to `nrf_cloud_agps` ?

There is a dependency to `nrf_cloud_pgps` in this module, but we intend to remove that.

Reply:
No.  The whole point of making `nrf_cloud_agps_configure_filter` was to eliminate redundancy, as Justin pointed out in his review comments.

I could consider moving it to some other library, I suppose.  It's not directly related to A-GPS, but it is a strange enough function that it is not useful for many other purposes.

Reply:
Ok. Then it doesn't make much sense to have
CONFIG_GPS_MODULE_AGPS_FILTERED and CONFIG_GPS_MODULE_ELEVATION_MASK. If this functionality is especially tied up against nRF Cloud A-GPS.